### ISSUE TITLE
Filter "parser": Incorrect value logged when invalid parser name given

### ISSUE THREAD

**Issue by fhoy** [fluent/fluent-bit#1008]
After https://github.com/fluent/fluent-bit/commit/0ae58c323a7e409ab7b942d443194467d2d4f9e9, the `tmp` variable used in the log statement in https://github.com/fluent/fluent-bit/blob/7c73ef5c9bb92dbfdad2405eb95501c7d412aa9d/plugins/filter_parser/filter_parser.c#L125 no longer contains the parser name. My understanding is that `p->val` should be used instead.

**To Reproduce**
Create a `[FILTER]` entry of type `parser`, with the `parser` field pointing to a non-existing parser. 

E.g.
```
./fluent-bit -i dummy -p "rate=10" -F parser -p "match=*" -p "key_name=log" -p "parser=foo" -o stdout
```
gives
`[2019/01/03 08:54:34] [error] [filter_parser] requested parser 'log' not found`
instead of the expected
`[2019/01/03 08:54:34] [error] [filter_parser] requested parser 'foo' not found`

**Comment by bluebike** [comment]
Had a look.... like this:
**plugins/filter_parser/filter_parser.c**
```c
          flb_error("[filter_parser] requested parser '%s' not found", p->val);
          return -1;
 ```

**Comment by edsiper** [comment]
thanks for the report and solution. Fixed by 6f837522

**Comment by edsiper** [comment]
fyi: fix to be included in v1.0.2 release (today).
